src/content/docs/tutorial-external-services/solve-external.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Solve your external service problem
+metaDescription: Resolve the root cause of your API or external service problems   
+---
+
+import apmPreview from 'images/apm_screenshot-crop_maps-preview.webp'
+
+import apmTrace from 'images/apm_screenshot-crop_maps-trace.webp'
+
+You have now identified the API or external service that requires attention. This tutorial guides you through the steps required to remediate the problems. 
+
+Note that there is no one solution for problems with APIs and external services. Take the ideas in this tutorial as a jumping off point and dig in whenever you uncover something.
+
+## Objectives
+
+This doc uses `Sms notification`, the external service we identified [in the previous tutorial doc](docs/tutorial-external-services/respond-external), as an example of a problematic API or external service. This doc covers:
+
+* Looking for anomalies in metric charts
+* Using traces to find the root cause
+
+## Identify and solve the root cause
+
+You've already used the map view to identify the problematic external service, but we can use it to go even deeper:
+
+<Steps>
+    <Step>
+### Identify anomalies with metrics
+
+In the map view hover over the problematic API or external service and select **Show entity preview**. 
+
+<img
+    title="The preview pane for an external service"
+    alt="The preview pane for an external service"
+    src={apmPreview}
+    />  
+
+
+This opens up a pane where you get an overview of the general state of the service. You already used some of these metrics to identify it as the problem, but it's useful to take a deeper look here. Keep an eye out for:
+
+* Any spikes in the charts. 
+* Obviously high response rate or throughput.
+* If response rate and throughput hit highs and lows around the same time.
```

Comment:
```suggestion
* Do response rate and throughput hit highs and lows around the same time?
```

Reply:
Not sure if this question works with the phrase `Keep an eye out for:` but it feels like it flows better. It's up to you!